spec/lib/tasks/email_missed_reservations_rake_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper'
+
+# see http://robots.thoughtbot.com/test-rake-tasks-like-a-boss
+describe 'email_missed_reservations' do
+  include_context 'rake'
+
+  before(:all) { FactoryGirl.create(:app_config) }
+
+  before(:each) do
+    @missed = FactoryGirl.create(:missed_reservation,
+                                 start_date: Date.current - 2.days,
+                                 due_date: Date.current - 1.days)
+    @not_missed = FactoryGirl.create(:valid_reservation)
+  end
+
+  it 'sends emails for approved reservations that were missed' do
+    @missed.approval_status = 'approved'
```

Comment:
You can shorten this, if you want, by using the `update` method (formerly `update_attributes` in Rails 3:

```
@missed.update(approval_status: 'approved')
```

This will run all of your validations, whereas `update_attribute` (note the singular) will not